src/checkNumbers.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-console */
+'use strict';
+
+const checkNumbers = (
+  terminal, generatedArr
+) => terminal.question('Guess number: ', (number) => {
+  if (String(number).length !== 4) {
```

Comment:
Also add check that the input is a number, currently the game considers `asdf` a valid number :)